controllers/news.js

```diff
@@ -49,8 +50,20 @@ exports.patchArticleVotes = (req, res, next) => {
     .then((article) => res.status(200).send({ article }))
     .catch(next);
 };
+
 exports.getUsers = (req, res, next) => {
   selectUsers().then((users) => {
     res.status(200).send({ users });
   });
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+  selectArticleById(article_id)
```

Comment:
you don't need this here, you're not doing anything with the result of it. Just do the post and it will be fine! :)